src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,308 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `service::pipelines::metrics`
+      </td>
+      <td>
+        Make sure `hostreceiver` and `resourcedetection` are included.
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+Here is a sample configuration YAML file:
+
+```
+extensions:
+  health_check:
+
+receivers:
+  hostmetrics:
+    collection_interval: 20s
+    scrapers:
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      load:
+      memory:
+        metrics:
+          system.memory.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      network:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      processes:
+      process:
+
+processors:
+  memory_limiter:
+    check_interval: 1s
+    limit_mib: 1000
+    spike_limit_mib: 200
+  batch:
+  cumulativetodelta:
+    metrics:
+      - system.network.io
+      - system.disk.operations
+      - system.network.dropped
+      - system.network.packets
+      - process.cpu.time
+  resource:
+    attributes:
+      - key: host.id
+        from_attribute: host.name
+        action: upsert
+  resourcedetection:
+    detectors: [ env, system]
+
+exporters: 
+  otlp:
+    endpoint: otlp.nr-data.net:4317
+    headers:
+      api-key: *****INGEST_LICENSE_KEY*****
+
+service:
+  pipelines:
+    metrics:
+      receivers: [hostmetrics]
+      processors: [batch, resourcedetection, resource, cumulativetodelta]
+      exporters: [otlp]
+
+  extensions: [health_check]
+```
+
+## Browsing host data in infrastructure UI [#using-ui]
+
+Using the recommended configuration for the host receiver enables all the standard capabilities of the [Infrastructure UI](/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui-entities#access-new-ui) (New Host UI) experience.
+
+## Query host metrics [#query-host-metrics]
+
+Once metrics are successfully ingested in New Relic, they are available in data explorer and query builder.
+
+The following NRQL queries show examples to help you explore the metrics you received:
+
+  * Listing number of metric updates ingested by metric name
+
+    ```
+    SELECT count(*) FROM Metric WHERE metricName LIKE 'system.%' FACET metricName LIMIT max
+    ```
+
+  * Querying specific metrics faceted by host
+
+    ```
+    SELECT average(system.disk.operations) FROM Metric FACET host.name TIMESERIES
+    ```
+
+  * Listing dimensions available for a given metric
+
+    ```
+    SELECT keyset() FROM Metric WHERE metricName = 'system.disk.operations' 
+    ```
+
+Learn more about [querying the metric data type](/docs/data-apis/understand-data/metric-data/query-metric-data-type).
+
+## Additional settings [#additional-settings]
```

Comment:
Starting from here till the end, I'd move it to a different doc focused on general settings for the collector & New Relic

Reply:
@josemore, I moved this into its own topic.